modules/promo-code/promo-code.service.js

```diff
@@ -1,28 +1,31 @@
 const RuleError = require('../../errors/rule.error');
 const PromoCode = require('./promo-code.model');
-
 const {
   STATUS_CODES: { NOT_FOUND },
 } = require('../../consts/status-codes');
 const {
   PROMOCODE_NOT_FOUND,
-} = require('../../error-messages/promocode.messages');
+} = require('../../error-messages/promocode-messages');
 
 class PromoCodeService {
   async getAllPromoCodes() {
```

Comment:
pagination for this?

Reply:
pagination will be implemented in another pull request

Reply:
why? its just few lines

Reply:
ok, create new PR for this case